src/main/java/org/acme/infinispanclient/model/HPMagic.java

```diff
@@ -0,0 +1,62 @@
+package org.acme.infinispanclient.model;
+
+import java.util.Objects;
+
+/**
+ * Author and Spell magic object
+ */
+public class HPMagic {
+   private final String id;
+   private final String author;
```

Comment:
Caster might be a better term?

Reply:
It's the first time I heard caster ... 🙈

Reply:
I didn't change to caster in the end ...

Reply:
Okay, I won't let this stop integration, but author doesn't seem like a proper usage for something like this. It sounds more like the person who made up the spell in the first place, not the person who is currently casting/using it.

Reply:
hmmm ok, I will use caster then